addons/website/static/src/components/views/page_list.js

```diff
@@ -14,6 +15,7 @@ export class PageListController extends PageControllerMixin(listView.Controller)
      */
     setup() {
         super.setup();
+        this.dialog = useService('dialog');
```

Comment:
It looks like the listView.Controller already defines `this.dialogService = useService('dialog');`, maybe it could be reused like in the `getActionMenuItems` override ?

Reply:
Indeed, removed